cernopendata/modules/fixtures/data/terms/terms.json

```diff
@@ -12746,5 +12746,186 @@
     "type": {
       "primary": "Glossary"
     }
+  },
+  {
+    "anchor": "CERNLIB",
+    "category": "generic",
+    "definition": "CERN Program library. The CERN Program Library is a large collection of general purpose libraries and modules maintained and offered in both source and object code form on the CERN central computers. The two most popular applications based on CERNLIB are PAW and GEANT 3.21. Most of these programs were developed at CERN and are therefore oriented towards the needs of a physics research laboratory that is general mathematics, data analysis, detectors simulation, data-handling etc... applicable to a wide range of problems. ",
+    "links": [
+      {
+        "text": "CERNLIB home page",
+        "url": "https://cernlib.web.cern.ch/cernlib/"
+      }
+    ],
+    "see_also": [
+      {
+        "term": "LEP"
+      },
+      {
+        "term": "DELPHI"
+      }
+    ],
+    "term": [
+      "CERNLIB"
+    ],
+    "type": {
+      "primary": "Glossary"
+    }
+  },
+  {
+    "anchor": "LEP",
+    "category": "generic",
+    "definition": "The Large Electron-Positron Collider",
+    "links": [
+      {
+        "text": "LEP page at CERN",
+        "url": "https://home.cern/science/accelerators/large-electron-positron-collider"
+      }
+    ],
+    "see_also": [
+      {
+        "term": "ALEPH"
+      },
+      {
+        "term": "DELPHI"
+      },
+      {
+        "term": "L3"
+      },
+      {
+        "term": "OPAL"
+      }
+    ],
+    "term": [
+      "LEP"
+    ],
```

Comment:
In this one, I would add "experiment" : ["ALEPH", "DELPHI", "L3", "OPAL"]. That way, we would have the small boxes with the names of the experiments, like in this screenshot.

It might be good to add them also to any other glossary term that applies to them


![Screenshot 2024-06-28 at 17 17 26](https://github.com/cernopendata/opendata.cern.ch/assets/12909623/8215260a-944c-4a8d-b99d-867e8e326c7c)